### PR TITLE
Release version 1.0.1

### PR DESCRIPTION
**Pull request by haleybash-optimizely**
@optimizely/fullstack-devs

### CODE REVIEW

**Comment by vraja2** [comment]
changelog should include more details about the release right?

File: CHANGELOG [hunk 1]
```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------------------
+1.0.1
+-------------------------------------------------------------------------------
+* Fix bug so conversion event is not sent if user is not bucketed into the experiment.
```
Comment (aliabbasrizvi) [inline CHANGELOG]: nit. Change to any experiment.

File: CHANGELOG [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+* Bump bluebird version from 3.3.5 to 3.4.6.
+* Update event endpoint.
```
Comment (aliabbasrizvi) [inline CHANGELOG]: nit. Can be more descriptive here.

**Comment by vraja2** [comment]
LGTM

**Pull request merged**